src/commands/Tools/Websearch/igdb.ts

```diff
@@ -58,7 +77,7 @@ export default class extends RichDisplayCommand {
 						...this.igdbRequestHeaders,
 						Authorization: `Bearer ${await this.client.twitch.fetchBearer()}`
 					},
-					body: this.buildBody(game)
+					body: this.commonQuery.concat(`search: "${game}"`)
```

Comment:
```suggestion
					body: `search: "${game}"; ${this.commonQuery}`
```

Code is invalid otherwise. You were calling String.Concat and not Array.Concat because of the `join('; ')`.